crates/bevy_gltf/src/loader.rs

```diff
@@ -479,14 +493,23 @@ async fn load_gltf<'a, 'b, 'c>(
                 && primitive.material().normal_texture().is_some()
             {
                 bevy_log::debug!(
-                    "Missing vertex tangents, computing them using the mikktspace algorithm"
+                    "Missing vertex tangents for {}, computing them using the mikktspace algorithm. Use Blender to bake this information into the gltf", file_name
```

Comment:
```suggestion
                    "Missing vertex tangents for {}, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.", file_name
```

I'll leave it to the maintainers when it comes to deciding if mentioning a tool is too opinionated but for me this is ok.

Reply:
@alice-i-cecile if you have a few free cycles please let us know about your opinion!

Reply:
Happy to endorse blender here.